PointOfDispensingApp/src/components/VaccinationEventPage.vue

```diff
@@ -10,11 +10,11 @@
     </v-row>
     <v-row>
       <PatientInfoComponent />
-      <VaccinationScreeningComponent />
-      <template v-if="true">
+      <VaccinationScreeningComponent @VaccinationProceed_Yes="setVaccinationProceedDecision(true)" @VaccinationProceed_No="setVaccinationProceedDecision(false)"/>
+      <template v-if="wasDecisionMadeToProceed()">
       <VaccinationProceedComponent />
       </template>
-      <template v-if="false">
+      <template v-if="wasDecisionMadeToCancel()">
```

Comment:
Can you wrap this and line 14's conditional to be if/else? If `wasDecisionMadeToProceed()` is false, it seems as if `wasDecisionMadeToCancel()` would be true. Or are they mutually exclusive?

Reply:
I updated line 17 to be "v-else-if"; the "else" case (there has not been a decision made to proceed with or cancel the vaccination) should not display any component.  In other words:
IF "decision was made to proceed with vaccination": display VaccinationProceedComponent.vue
ELSE-IF "decision was made to cancel vaccination": display VaccinationCanceledComponent.vue
ELSE (no decision was made): display nothing